libs/utils/utils.js

```diff
@@ -849,13 +849,18 @@ export async function loadArea(area = document) {
   if (isDoc) {
     const georouting = getMetadata('georouting') || config.geoRouting;
     if (georouting === 'on') {
+      // eslint-disable-next-line import/no-cycle
       const { default: loadGeoRouting } = await import('../features/georoutingv2/georoutingv2.js');
       loadGeoRouting(config, createTag, getMetadata, loadBlock, loadStyle);
     }
     const appendage = getMetadata('title-append');
```

Comment:
Since this `getMetadata`-> loadFeature pattern is repeated a few times now - should we think about having this a bit more generic?

Something such as `metafeature-seotech-video-url` and everything that's prefixed as "metafeature" will try to load the feature `seotech-video-url`

Maybe a bit outside of the scope of this PR but we could shave a few LOC of the utils and future-proof this. Wdyt? //cc @chrischrischris

Reply:
I noticed this as well but it's out of scope.

I would wait until this area gets bigger. I think it's too early to create an abstraction here. For example, in some cases more than one flag (feature on + feature data) could trigger the branch.